env.template

```diff
@@ -46,8 +46,7 @@ AIRFLOW_CONN_AWS_DEFAULT=aws://test_key:test_secret@?region_name=us-east-1&host=
 # Change the following line in prod to use the appropriate DB
 AIRFLOW_CONN_POSTGRES_OPENLEDGER_UPSTREAM=postgres://deploy:deploy@postgres:5432/openledger
 AIRFLOW_CONN_POSTGRES_OPENLEDGER_TESTING=postgres://deploy:deploy@postgres:5432/openledger
-# Slack webhook connection info (this must be URL encoded, with a
-# pre-pended https://, e.g. "https://https%3A%2F%2Fhooks.slack.com%2Fservices%2everythingelse")
+# Slack webhook connection info
```

Comment:
An extra note here might be good in case people super familiar with Airflow get tripped up by it:
```suggestion
# Slack webhook connection info (note that these values are modified by the Docker entrypoint)
```